content/src/view.js

```diff
@@ -2208,44 +2208,15 @@ function changeEditorText(paneState, text) {
   if (text.length && text.charAt(text.length - 1) != '\n') {
     text += '\n';
   }
-  paneState.changeHandler.suppressChange = true;
-  var saved = {}, editor = paneState.editor, session = editor.session;
-  saved.selection = session.selection.toJSON()
-  saved.atend = session.selection.getCursor().row >= session.getLength() - 1;
-  saved.folds = session.getAllFolds().map(function(fold) {
```

Comment:
this method had not used anyway according to my knowledge. And also it seems like a copy of another method. So, I saw them as useless things. Anyway, I'l check it again and let you know